types/react/test/index.ts

```diff
@@ -228,6 +229,7 @@ const notValid: boolean = React.isValidElement(props); // false
 const isValid = React.isValidElement(element); // true
 let domNode: Element = ReactDOM.findDOMNode(component);
 domNode = ReactDOM.findDOMNode(domNode);
+const fragmentType: symbol | number = React.Fragment;
```

Comment:
I'm of the opinion that the actual type of `React.Fragment` should be opaque to clients. The fact that it is a `symbol | number` is an implementation detail. As @tkrotoff suggested I would feel better about changing references to an alias like `type FragmentType = symbol & 0xeacb;`

Reply:
Unfortunately, our type aliases aren't opaque. So creating something like `type FragmentType = symbol | 0xeacb;` wouldn't be visible by clients anyway.

Reply:
All I'm saying is this test shouldn't be testing that `React.Fragment` is a `symbol | number`, because that's liable to change. Something like

```ts
const fragmentType: React.FragmentType = React.Fragment
```

would be appropriate.

Reply:
A type like `symbol & 0xeacb` prevents users from making a mistake by passing a `number` into `createElement`.